tests/test_wave_05.py

```diff
@@ -80,34 +81,37 @@ def test_create_goal(client):
     }
 
 
-@pytest.mark.skip(reason="test to be completed by student")
+#@pytest.mark.skip(reason="test to be completed by student")
 def test_update_goal(client, one_goal):
-    raise Exception("Complete test")
+    #raise Exception("Complete test")
     # Act
     # ---- Complete Act Here ----
-
+    client.put("/goals/1",json={"title": "Never leave the house again"})
+    response = client.get("/goals/1")
+    response_body = response.get_json()
     # Assert
     # ---- Complete Assertions Here ----
-    # assertion 1 goes here
-    # assertion 2 goes here
-    # assertion 3 goes here
+    assert response.status_code == 200
+    assert "goal" in response_body
+    assert "Never leave the house again" == response_body["goal"]["title"]
```

Comment:
Definitely not incorrect, but a bit more common to put the result on the left side of the assert identity and the expected item to be on the right.